project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 
 resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
 
 addSbtPlugin("com.storm-enroute" % "sbt-lwjgl-plugin" % "3.1.6")
+//addSbtPlugin("com.github.philcali" % "sbt-lwjgl-plugin" % "3.1.6")
```

Comment:
This was not published on maven under `com.github.philcali`, as far as I am aware of, so we can remove this comment, and just use the `com.storm-enroute` dependency that I published some time ago.